code/modules/admin/verbs/debug.dm

```diff
@@ -301,33 +301,65 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 	else
 		alert("Invalid mob")
 
-/client/proc/cmd_admin_change_hivenumber(mob/living/carbon/Xenomorph/M in mob_list, var/hivenumber = XENO_HIVE_NORMAL)
+/client/proc/cmd_admin_change_hivenumber(mob/living/carbon/Xenomorph/X in mob_list)
 	set category = "Debug"
 	set name = "Change Hivenumber"
 
+	if(!check_rights(R_DEBUG|R_ADMIN))	
+		return
+
+	if(!istype(X))
+		to_chat(usr, "This can only be done to instances of type /mob/living/carbon/Xenomorph")
+		return
+
+	var/hivenumber_status = X.hivenumber
+	var/list/namelist = list("Normal","Corrupted","Alpha","Beta","Zeta")
```

Comment:
no, bad (yes i know this is my code shut up i know better now)
```suggestion
	var/list/namelist = list()
	for(var/datum/hive_status/H in hive_datum) // global hive datum list
		namelist += H.name
```
Also you'll need to add a name var to `/datum/hive_status`